CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## master
 
+## 0.6.0
```

Comment:
Please leave it to `master` branch. We should change the CHANGELOG version only upon release.

Reply:
Sounds good.

How do we reflect the changes we are introducing without forgetting any though? Wouldn't it make sense to add the changes regardless? Maybe without the version number as you pointed out.

Reply:
Yes the changes should/can stay. We should just leave them in the `## master` section and not assign a version yet